### PR TITLE
Update zh_cn translate

### PR DESCRIPTION
**Pull request by Parallel011**
Hello! I made some Chinese translation. it's not completed yet, and I'm not sure if what I'm doing is the right thing.
But I'm sure that it's working correctly.

### CODE REVIEW

**Comment by nonamecrackers2** [comment]
Looks good, should complete it before I put it in

**Comment by Parallel011** [comment]
All the translation have been already completed.

**Comment by nonamecrackers2** [comment]
Looks good, I'll try to get it added in the next update

File: lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+	"advancements.witherstormmod.main.silver_lining.description": "制作一把命令工具，这种工具充满命令方块的强大功能",
+	"advancements.witherstormmod.main.insane_dedication.title": "近乎可怕的奉献",
+	"advancements.witherstormmod.main.insane_dedication.description": "制作一把命令尖锄，然后回头看看是什么让你走到了这一步",
+	"advancements.witherstormmod.main.escape_wither_storm.title": "死亡如风",
+	"advancements.witherstormmod.main.escape_wither_storm.description": "击打凋灵风暴的头部，逃离它的魔掌",
+	"advancements.witherstormmod.main.cured_sickened_mob.title": "职业医师",
+	"advancements.witherstormmod.main.cured_sickened_mob.description": "用金苹果炖菜治愈一只被腐化的生物",
+	"advancements.witherstormmod.main.belly_of_the_beast.title": "野兽之内……",
+	"advancements.witherstormmod.main.belly_of_the_beast.description": "进入凋灵风暴的内部空间",
+	"advancements.witherstormmod.main.wither_storm_defeated.title": "故事永不结束",
```
Comment (Brian01071) [inline lang/zh_cn.json:277]: （我记得故事模式中有原句好像可以对应？）

Comment (nonamecrackers2) [reply lang/zh_cn.json:277]: English?

**Comment by nonamecrackers2** [comment]
There's a few things that are missing, so I'll have to mark this translation as mostly complete. If you could (or anyone else) finish everything that would be great, however no pressure.

**Pull request merged**